api/checks/webhook.go

```diff
@@ -338,9 +338,7 @@ func isUserWhitelisted(aeAPI shared.AppEngineAPI, login string) bool {
 		return true
 	}
 	whitelist := []string{
-		"autofoolip",
 		"chromium-wpt-export-bot",
-		"foolip",
 		"gsnedders",
```

Comment:
@gsnedders @jgraham @jugglinmike @lukebjerring @Ms2ger FYI I'm opting myself out of staging checks. If anyone else doesn't want to see the extra checks either I can roll that into this PR.